### PR TITLE
#1584 Implementing nextafter for float and double

### PR DESCRIPTION
**Pull request by pdet**
This PR exposes the nextafter and nextafterf from c to SQL

Testing it through the SQLTest API is a bit tricky, due to the precision difs in the next after for float and double, so let me know if anyone has a better idea than what I did there :-)

### CODE REVIEW

File: test/sql/function/numeric/test_nextafter.test [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+statement error
+select nextafter('bla','bla')
+
+statement error
+select nextafter(99, 1)
+----
+True
+
+query R
+select nextafter(99.0::DOUBLE, 1.0::DOUBLE) < 99
```
Comment (Mytherin) [inline test/sql/function/numeric/test_nextafter.test:20]: Could you add a test with NULL, and a test with the two values being equal?

**Review by Mytherin — COMMENTED** [review]
Thanks for the PR! Looks good. Some comments:
